README.md

```diff
@@ -54,7 +54,8 @@ You can provide additional parameters to constructor:
 - `url` - by default empty and **REQUIRED**.
 - `userAgent` - by default `Mozilla`. You can set userAgent from request.
 - `keepListen` - by default `false`. If you set to `true`, then response from radio station is not destroys and you can pipe it to another streams.
-- `autoUpdate` - by default `true`. If you set to `false`, then metadata will not be updates with interval and notify you about new metadata;
+- `autoUpdate` - by default `true`. If you set to `false`, then metadata will not be updates with interval and notify you about new metadata.
+- `notifyOnChangeOnly` - by default `false`. If you set to `true`, then metadata will be checked with interval and notify you about new metadata but the channel will not close as it does in `autoUpdate`.
```

Comment:
Both mine and yours need to be rephrased now :)
It is not obvious what exactly it means and what is the difference.

What about:

> `autoUpdate` - by default `true`. If you set to `false`, then parser will not be listening for recent updates and immediately close the stream. So you will get a metadata only once.
> `notifyOnChangeOnly` - by default `false`. If you set both `autoUpdate` and `notifyOnChangeOnly` to `true`, it will keep listening the stream and notifying you about metadata, but it will not notify if metadata did not change from the previous time.